assets/js/modules/analytics/components/dashboard/AnalyticsAdSenseDashboardWidgetLayout.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * AnalyticsAdSenseDashboardWidgetLayout component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as MODULES_ADSENSE } from '../../../adsense/datastore/constants';
+import { STORE_NAME as MODULES_SEARCH_CONSOLE } from '../../../search-console/datastore/constants';
+import Layout from '../../../../components/layout/layout';
+
+const { useSelect } = Data;
+
+const AnalyticsAdSenseDashboardWidgetLayout = ( { children } ) => {
+	const adSenseAccountID = useSelect( ( select ) => select( MODULES_ADSENSE ).getAccountID() );
+	const propertyURL = useSelect( ( select ) => select( MODULES_SEARCH_CONSOLE ).getPropertyID() );
+	const accountURL = useSelect( ( select ) => select( MODULES_ADSENSE ).getServiceURL(
+		{
+			path: `/${ adSenseAccountID }/home`,
+			query: {
+				source: 'site-kit',
+				url: propertyURL,
+			},
+		}
+	) );
```

Comment:
@felixarntz this is very similar to the getAccountURL selector we created with the exception of adding the URL - should we perhaps create a selector for this use-case?

Reply:
@ryanwelcher I thought about this too. Maybe this would make sense as `getAccountSiteURL`? We already have a function called that, but the URL from that is for managing a site, while the one here is more just the overview of that site. I think what would make sense is:

1. Rename the current `getAccountSiteURL` to `getAccountManageSiteURL` and update all references.
2. Rename the new `getAccountSitesURL` to `getAccountManageSitesURL` and update all references.
3. Introduce `getAccountSiteURL` for the above usage.

What do you think?

Reply:
@tofumatt For context, this is how I suggested changing selector names to more appropriately reflect what these URLs are for.

Reply:
This still hasn't been updated, we should introduce `getAccountSiteURL` (or now `getServiceAccountSiteURL`).